x-pack/test/case_api_integration/spaces_only/tests/common/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ loadTestFile }: FtrProviderContext): void => {
+  describe('Common', function () {
+    loadTestFile(require.resolve('./comments/delete_comment'));
+    loadTestFile(require.resolve('./comments/find_comments'));
+    loadTestFile(require.resolve('./comments/get_comment'));
+    loadTestFile(require.resolve('./comments/get_all_comments'));
+    loadTestFile(require.resolve('./comments/patch_comment'));
+    loadTestFile(require.resolve('./comments/post_comment'));
+    loadTestFile(require.resolve('./cases/delete_cases'));
+    loadTestFile(require.resolve('./cases/find_cases'));
+    loadTestFile(require.resolve('./cases/get_case'));
+    loadTestFile(require.resolve('./cases/patch_cases'));
+    loadTestFile(require.resolve('./cases/post_case'));
+    loadTestFile(require.resolve('./cases/reporters/get_reporters'));
+    loadTestFile(require.resolve('./cases/status/get_status'));
+    loadTestFile(require.resolve('./cases/tags/get_tags'));
+    loadTestFile(require.resolve('./user_actions/get_all_user_actions'));
+    loadTestFile(require.resolve('./configure/get_configure'));
+    loadTestFile(require.resolve('./configure/patch_configure'));
+    loadTestFile(require.resolve('./configure/post_configure'));
```

Comment:
Do you think it doesn't make sense to test subcases or the case connector as they are disabled?

Reply:
I chatted with Xavier and he recommended not spending time implementing the RBAC for case connector because we aren't able to test the behavior without changes to the actions framework and we haven't completed the case as a connector feature. So I think it's probably ok to not test them here.